### PR TITLE
Issue #13213: Removed '//ok' from javadocparagraph

### PR DESCRIPTION
**Pull request by 555vedant**
* Issue #13213

### CODE REVIEW

**Comment by romani** [comment]
Change commit message to be single line

**Comment by 555vedant** [comment]
Is this fine now?

**Comment by 555vedant** [comment]
how can i fix this pending checks ?

**Review by MANISH-K-07 — COMMENTED** [review]
@romani , @nrmancuso 

`FYI :` This PR might raise conflicts with PR #14551  while merge

File: config/checkstyle-input-suppressions.xml
Comment (555vedant) [reply config/checkstyle-input-suppressions.xml:877]: so whats the solution now tell me that thing ?

Comment (555vedant) [reply config/checkstyle-input-suppressions.xml:877]: @MANISH-K-07 is this pr ready to merge ?

Comment (MANISH-K-07) [reply config/checkstyle-input-suppressions.xml:877]: Please wait for @romani 's approval.

**Review by nrmancuso — APPROVED** [review]
Ok to merge as GitHub CI passes

**Comment by nrmancuso** [comment]
@555vedant thanks for your contribution!

**Comment by 555vedant** [comment]
Sir now my work is done?

**Comment by 555vedant** [comment]
@nrmancuso  Sir now my work is done?

**Comment by nrmancuso** [comment]
@555vedant please resolve conflicts

**Comment by 555vedant** [comment]
> @555vedant please resolve conflicts
please suggest me how can i fix that conflicts

**Comment by 555vedant** [comment]
@nrmancuso  please suggest me how can i fix that conflicts?

**Comment by 555vedant** [comment]
@romani  done

File: config/checkstyle-input-suppressions.xml
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:847]: @555vedant ,
The whole point of the issue is to pick a suppression and resolve it.

**Review by MANISH-K-07 — COMMENTED** [review]
Please revert your changes....

**Comment by nrmancuso** [comment]
@555vedant looks like javadocparagraph suppressions were already removed in a concurrent PR. Let's close this one and pick some new suppressions to work on.

**Pull request closed without merging**
